refinery/ui/source/js/dashboard/controllers/primary-group-button-ctrl.js

```diff
@@ -13,11 +13,13 @@
 
```

Comment:
the child component is responsible for loading the initial dashboard's data sets